akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentBehaviorSpec.scala

```diff
@@ -140,30 +140,30 @@ object PersistentBehaviorSpec {
         case IncrementTwiceAndThenLog ⇒
           Effect
             .persist(Incremented(1), Incremented(1))
-            .andThen {
+            .andThen { (_: State) ⇒
```

Comment:
Needed here as well?

Reply:
yes, compiler says so:
```
[error] /Users/patrik/dev/akka/akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentBehaviorSpec.scala:163:24: missing parameter type
[error]             .andThen { _ ⇒
[error]                        ^
```

If it wasn't for the two `.andThen` it wouldn't be needed here.

One would never write code like this, but we are exercising the edge cases.